go/enclave/enclave.go

```diff
@@ -227,20 +228,35 @@ func (e *enclaveImpl) SubmitRollup(rollup common.ExtRollup) {
 	}
 }
 
-func (e *enclaveImpl) SubmitTx(tx common.EncryptedTx) error {
+func (e *enclaveImpl) SubmitTx(tx common.EncryptedTx) (common.EncryptedResponseSendRawTx, error) {
 	decryptedTx, err := e.rpcEncryptionManager.DecryptTx(tx)
 	if err != nil {
-		return fmt.Errorf("could not decrypt transaction. Cause: %w", err)
+		log.Info(fmt.Sprintf("could not decrypt transaction. Cause: %s", err))
```

Comment:
Yeah fair, not a big deal anyway. Imagine we might end up logging the errors in rpc_server so that we can control it more easily for all of them eventually.